### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by pchtsp**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -88,19 +88,19 @@ You can get the value of the variables using value(). ex::
...
+*  ``LpConstraintVar`` -- Used to construct a column of the model in column-wise modelling
 
 Exported Functions:
 
-* `value()` -- Finds the value of a variable or expression
-* `lpSum()` -- given a list of the form [a1*x1, a2x2, ..., anxn] will construct a linear expression to be used as a constraint or variable
-* `lpDot()` --given two lists of the form [a1, a2, ..., an] and [ x1, x2, ..., xn] will construct a linear epression to be used as a constraint or variable
+* ``value()`` -- Finds the value of a variable or expression
+* ``lpSum()`` -- given a list of the form [a1*x1, a2x2, ..., anxn] will construct a linear expression to be used as a constraint or variable
+* ``lpDot()`` --given two lists of the form [a1, a2, ..., an] and [ x1, x2, ..., xn] will construct a linear epression to be used as a constraint or variable
```
Comment (afifabroory) [inline README.rst:103]: It's typo? "linear epression" and it should "linear expression" right?

Comment (afifabroory) [inline README.rst:103]: > ``lpDot()`` --given two lists of the form [a1, a2, ..., an] and [ x1, x2, ..., xn] will construct a linear epression to be used as a constraint or variable

Is this a typo? "linear epression" should be "linear expression" right?

Comment (pchtsp) [reply README.rst:103]: Yep. Feel free to do a PR and I'll merge it.

Comment (afifabroory) [reply README.rst:103]: Okay, thanks

**Pull request merged**